als-common/shared/src/main/scala/org/mulesoft/amfintegration/AmfImplicits.scala

```diff
@@ -315,8 +314,9 @@ object AmfImplicits {
   }
 
   implicit class DialectInstanceImp(instance: DialectInstance) {
+    private val validationProfileDialectId: String = "http://a.ml/amf/default_document/validationprofile/1.0"
```

Comment:
Regarding this particular line, it would be convenient for this value to be provided directly by the ValidationProfileDialectLoader in the future so we can delete this.

Reply:
@pope1838 any chance to set this ID in the object so we don't have to wait for a future or hardcode this string?